auth/oidc.go

```diff
@@ -217,27 +217,27 @@ func (op *OIDCProvider) FetchCustomProviderConfig(discoveryURL string) (*oidc.Pr
 		discoveryURL = strings.TrimSuffix(op.Issuer, "/") + discoveryConfigPath
 	}
 
-	base.LogTo("OIDC+", "Fetching custom provider config from %s", discoveryURL)
+	base.LogToR("OIDC+", "Fetching custom provider config from %s", discoveryURL)
 	req, err := http.NewRequest("GET", discoveryURL, nil)
 	if err != nil {
-		base.LogTo("OIDC+", "Error building new request for URL %s: %v", discoveryURL, err)
+		base.LogToR("OIDC+", "Error building new request for URL %s: %v", base.UD(discoveryURL), err)
 		return nil, err
 	}
 	resp, err := http.DefaultClient.Do(req)
 	if err != nil {
-		base.LogTo("OIDC+", "Error invoking calling discovery URL %s: %v", discoveryURL, err)
+		base.LogToR("OIDC+", "Error invoking calling discovery URL %s: %v", base.UD(discoveryURL), err)
 		return nil, err
 	}
 	defer resp.Body.Close()
 	if err := json.NewDecoder(resp.Body).Decode(&customConfig); err != nil {
-		base.LogTo("OIDC+", "Error parsing body %s: %v", discoveryURL, err)
+		base.LogToR("OIDC+", "Error parsing body %s: %v", discoveryURL, err)
```

Comment:
Looks like the other cases of `discoveryURL` were wrapped in `base.UD()`

Reply:
Thanks, good spot!